tests/test_tensorflow_probability.py

```diff
@@ -0,0 +1,10 @@
+import unittest
+
+import tensorflow_probability as tfp
```

Comment:
In our current image, I am able to `import tensorflow_probability` directly, it only fails if I first `import tensorflow`.